src/games/BaselineTriangle/Triangle.h

```diff
@@ -28,19 +28,19 @@ struct GameData
 
     struct Shaders
     {
-        const char* VertexShader = "src/games/BaselineTriangle/shaders/set1/bin/VertexShader.vert.spv"; 
+        const char* VertexShader = "../src/games/BaselineTriangle/shaders/set1/bin/VertexShader.vert.spv"; 
 
-        const char* FragmentShader = "src/games/BaselineTriangle/shaders/set1/bin/FragmentShader.frag.spv";
+        const char* FragmentShader = "../src/games/BaselineTriangle/shaders/set1/bin/FragmentShader.frag.spv";
 
         // Tex coord passthrough VS
-        const char* VertexShader_Tex = "src/games/BaselineTriangle/shaders/set2/bin/VertexShader_Tex.vert.spv";
+        const char* VertexShader_Tex = "../src/games/BaselineTriangle/shaders/set2/bin/VertexShader_Tex.vert.spv";
 
-        const char* FragmentShader_Tex = "src/games/BaselineTriangle/shaders/set2/bin/FragmentShader_Tex.frag.spv"; 
+        const char* FragmentShader_Tex = "../src/games/BaselineTriangle/shaders/set2/bin/FragmentShader_Tex.frag.spv"; 
         
         // Tex coord passthrough VS
-        const char* VertexShader_Tex2 = "src/games/BaselineTriangle/shaders/set3/bin/VertexShader_Tex2.vert.spv"; 
+        const char* VertexShader_Tex2 = "../src/games/BaselineTriangle/shaders/set3/bin/VertexShader_Tex2.vert.spv"; 
 
-        const char* FragmentShader_Tex2 = "src/games/BaselineTriangle/shaders/set3/bin/FragmentShader_Tex2.frag.spv";
+        const char* FragmentShader_Tex2 = "../src/games/BaselineTriangle/shaders/set3/bin/FragmentShader_Tex2.frag.spv";
```

Comment:
This works for now. I will probably change this requirement of specifying the path like this later on. Note this is also sensitive to where the executable is called from (on Linux at least), so this will be something to consider for a v.1.0.0/packaging/different platforms.